promscale/installation/docker.md

```diff
@@ -52,6 +52,11 @@ packages and instructions, see the
 
 </procedure>
 
+<highlight type="note">
+Post installation of Promscale, the next step is to ingest the 
+data into Promscale. Follow the [send data][send-data] instructions.
+</highlight>
```

Comment:
```suggestion
When you have installed Promscale, you are ready to ingest 
data. For instructions, see the [send data][send-data] section.
```

Reply:
Just an additional note: reference-style links won't work inside an admonition. I haven't changed that, though, since I took it out of the admonition.